integration_test/tests/src/tests/HubManager.test.ts

```diff
@@ -0,0 +1,38 @@
+import { waitForReact } from 'testcafe-react-selectors';
+import { config } from '../config';
+import { LandingMenu } from '../page-objects/components/LandingMenu';
+import { HubManagerPage, DFSPRow } from '../page-objects/pages/HubManager';
+import chance from 'chance';
+import { Selector } from 'testcafe';
+
+fixture `Hub Management Feature`
+  .page`${config.connectionManagerEndpoint}`
+  .beforeEach(async (t) => {
+    await waitForReact();
+    await t
+      .click(LandingMenu.hubButton).wait(1000);
+  });
+
+test.meta({
+  ID: '',
+  STORY: '',
+  description: '',
+})('Able to add DFSP in DFSP Administration sub menu', async (t) => {
+  // Navigate to Add DFSP feature
+  await t.click(HubManagerPage.hubPageAdministrationDfspSubMenuButton).wait(1000);
+  await t.click(HubManagerPage.hubPageAdministrationDfspSubMenuAddDfspButton).wait(1000);
```

Comment:
Have you tried without the wait here? What's the default?

Reply:
Btw, maybe just add a comment explaining the wait, because if the test turns out to be a little brittle or something later on this is going to be the first place someone looks, and a comment would be useful.

Reply:
I always err adding them, but I know there is a global setting, I'll put that in a follow up PR.